### PR TITLE
OpenShift Calculation Update

### LINKED ISSUE #144: Update NERC documentation for Pricing based on updated OpenShift Calculation
## Motivation
An update to OpenShift invoice calculation requires an update to documentation.

## Completion Criteria
The [NERC doc page](https://nerc-project.github.io/nerc-docs/get-started/cost-billing/how-pricing-works/) includes information update from the [Invoice Requirements](https://docs.google.com/document/d/1SHCMidWQmazACGqzYOlRHQqnsg4KjmeW8ShGiEptR2Q/edit?pli=1&disco=AAABCX1bcpk) doc.

## Description
- [x] Update document
- [x] Create a PR
- [x] Link PR
- merge PR

## Completion dates
Desired - 2023-12-22
Required - 2024-01-02

### PR DESCRIPTION
**Pull request by joachimweyl**
A calculation update required a new example

Closes #144

### CODE REVIEW

File: docs/get-started/cost-billing/how-pricing-works.md [hunk 1]
```diff
@@ -49,6 +54,27 @@ of the base SU for the maximum resource they reserve.
 
     `$46.80`
 
+**OpenShift CPU SU Example:**
+
+- Project with 3 Pods with:
+
+     `1 vCPU, 3 GB RAM, 720hrs (24hr*30days)`
+     `.1 vCPU, 8 GB RAM, 720hrs (24hr*30days)`
```
Comment (naved001) [inline docs/get-started/cost-billing/how-pricing-works.md:62]: I prefer having leading zeros in decimals less than 1. So, I would write this as "0.1 vCPU".

File: docs/get-started/cost-billing/how-pricing-works.md [hunk 2]
```diff
@@ -49,6 +54,27 @@ of the base SU for the maximum resource they reserve.
...
+    `2 CPU SUs due to extra RAM (8GB vs .4GB(.1*4GB)) * 720hrs * $0.013`
+    `2 CPU SUs due to more CPU (2vCPU vs 1vCPU(4GB/4)) * 720hrs * $0.013))`
+    `=RoundUP(Sum(720(1+2+2)))*.013`
+
+    `$46.80`
+
+If you prefer a function for the OpenShift pods here it is:
+Project SU HR count = RoundUP(SUM(Pod1 SU hour count + Pod2 SU hr count + ...))
+
 OpenShift Pods are summed up to the project level so that fractions of CPU/RAM
```
Comment (naved001) [inline docs/get-started/cost-billing/how-pricing-works.md:78]: Does this need to be updated?

Comment (joachimweyl) [reply docs/get-started/cost-billing/how-pricing-works.md:75]: Not required, just going to remove it.

**Comment by joachimweyl** [comment]
@naved001 changes made, when you have a chance please re-review.

**Pull request merged**